package.json

```diff
@@ -94,7 +94,8 @@
 		"source-registry-storage-adapter": "github:devx-services/source-registry-storage-adapter#main",
 		"util": "^0.12.5",
 		"uuid": "^8.3.2",
-		"yaml": "^2.4.2"
+		"yaml": "^2.4.2",
+		"newrelic": "^11.0.0"
```

Comment:
Just to understand, it seems like we are adding newrelic package as a dependency of our CLI plugin such that the local mesh command will be able to run without error for meshes that include local hooks.

It feels odd to include this dependency in our CLI plugin since I don't think we would ever actually instrument the plugin with new relic nor is it actually in use.

Is this just a result of our local hooks instrumentation being so tightly coupled to the newrelic module that we cant exclude it when the module is not present?

Reply:
Yes thats true. This is a result of how the hooks package is laid out. Maybe upon our refactor for edge, we can remove that dependency to adjust in such a way that the CLI wouldn't need to provide newrelic. But for now we will need to add it.